### ISSUE TITLE
Can't be present at the same time

### ISSUE THREAD

**Issue by justLoveJ** [huri000/SwiftEntryKit#230]
Toast pops up at the top, and then there's no way to pop alert

**Comment by huri000** [comment]
If I understand correctly; multiple entries cannot be displayed in parallel using this library, but `SwiftEntryKit` offers other solutions. Duplicate of #88 if I understand correctly.
